frontend/src/dialogs/ProjectDialog.vue

```diff
@@ -192,28 +194,26 @@ export default {
       }
     },
     projectNames () {
-      return projectNamesFromProjectList()
+      return this.projectNamesFromProjectList
     },
     projectDetails () {
       return getProjectDetails(this.project)
     },
-    costObjectSettings () {
-      return getCostObjectSettings() || {}
-    },
     costObjectSettingEnabled () {
-      return getCostObjectSettings() !== undefined
+      return this.costObjectSettings !== undefined
```

Comment:
enabled if `false, null or ''`

Reply:
will change it to !isEmpty(this.costObjectSettings) as discussed